go/v4/exchange_orderbook.go

```diff
@@ -0,0 +1,414 @@
+package ccxt
```

Comment:
@caoilainnl I know this is a pain, but we will need to create a different package for the WS code for a couple of reasons:

- the current solution is already taking a huge amount of ram/cpu to build and by adding the WS section, it will get even worse
- REST-only users shouldn't be importing the WS code
- updating a ws-only section of the code shouln;'t require an entire build

We can temporarily keep this solution to make sure the transpiling/ws structs/etc are working as expected and then we do this upgrade before merging.